src/Event/BaseEvent.php

```diff
@@ -116,14 +127,23 @@ public function getVersion(): int
     }
 
     /**
-     * @return mixed[]
+     * @return string[]|int[]
```

Comment:
Are you sure this is correct? Looks like `string[] OR int[]`, where the content is `( string|int|mixed[] )[]`. Would just leave it `mixed` and let the psalm specification be the detailed version